fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/plans/RecordQueryIndexPlan.java

```diff
@@ -206,14 +208,39 @@ public int getComplexity() {
     @Nonnull
     @Override
     public PlannerGraph rewritePlannerGraph(@Nonnull final List<? extends PlannerGraph> childGraphs) {
-        final PlannerGraph.Node root =
-                new PlannerGraph.Node(this,
-                        getClass().getSimpleName(),
-                        getRange() + "; " + (scanType == IndexScanType.BY_VALUE ? scanType : "") + "; " + (reverse ? "reverse" : ""));
-        final PlannerGraph.SourceNode source = new PlannerGraph.SourceNode(getIndexName());
-        return PlannerGraph.builder(root)
-                .addNode(source)
-                .addEdge(source, root, new PlannerGraph.Edge())
-                .build();
+        final Optional<TupleRange> tupleRangeOptional = getTupleRangeOptional();
+
+        final ImmutableList.Builder<String> detailsBuilder = ImmutableList.builder();
+        final ImmutableMap.Builder<String, Attribute> additionalAttributes = ImmutableMap.builder();
+
+        if (scanType != IndexScanType.BY_VALUE) {
+            detailsBuilder.add("scanType: {{scanType}}");
+            additionalAttributes.put("scanType", Attribute.gml(scanType.toString()));
+        }
+
+        if (tupleRangeOptional.isPresent()) {
+            final TupleRange tupleRange = tupleRangeOptional.get();
+            detailsBuilder.add("range: " + tupleRange.getLowEndpoint().toString(false) + "{{low}}, {{high}}" + tupleRange.getHighEndpoint().toString(true));
+            additionalAttributes.put("low", Attribute.gml(tupleRange.getLow() == null ? "∞" : tupleRange.getLow().toString()));
```

Comment:
I think this should probably be `-∞` or something like that. Alternatively, we could use literals like "start of range" or "end of range" or something like that.

Reply:
done